src/pages/index.md

```diff
@@ -2,5 +2,19 @@
 title: About Fedi Devs
 id: about
 slug: /
-sidebar_position: 1
 ---
+
+# Fediverse Developer Network
+
+## What is it?
+
+- Build a community around those building for the Fediverse.
```

Comment:
I would suggest to be more specific and focus specifically on developers with active projects that want to enable their users to interoperate with other active projects. There are many other places where others can hang out -- like for people who like specs, or users, or people who have opinions without active codebases.